seed_data/lib.py

```diff
@@ -2,14 +2,13 @@
 Library functions for interacting with test data
```

Comment:
i'm happy to clean up this seed data code in a later PR. if some parts of it break as a result of this PR, there isn't a significant impact on us